doc/doxygen/Doxyfile.in

```diff
@@ -2457,7 +2457,7 @@ INTERACTIVE_SVG        = NO
 # found. If left blank, it is assumed the dot tool can be found in the path.
 # This tag requires that the tag HAVE_DOT is set to YES.
 
-DOT_PATH               =
```

Comment:
Hmm.. I think it would be better to leave it blank or configure it with the basename of \@DOXYGEN_DOT_EXECUTABLE\@

Reply:
whoops - yes of cause!